scraper.rb

```diff
@@ -11,13 +32,14 @@ def scrape_page(page, comment_url)
       "info_url" => (page.uri + tr.search('td').at('a')["href"]).to_s,
       "council_reference" => tds[1],
       "date_received" => Date.new(year, month, day).to_s,
-      "description" => tds[3].gsub("&amp;", "&").split("<br>")[1].squeeze(" ").strip,
+      "description" => tds[3].gsub("&amp;", "&").split("<br>")[1].to_s.squeeze(" ").strip,
```

Comment:
It's worth noting that applications without a description are not imported into PlanningAlerts.